model/change_point_detection.go

```diff
@@ -126,6 +128,17 @@ type PerformanceData struct {
 	Data                []MeasurementData         `bson:"data"`
 }
 
+type ChangePointWithPerformanceData struct {
+	PerformanceResultSeriesID `bson:",inline"`
```

Comment:
Can we avoid inlining? I think that locks us into the legacy bson library in a way that I'm not comfortable with.